.buildkite/pipeline.yml

```diff
@@ -91,10 +91,10 @@ steps:
       slurm_ntasks: 1
       slurm_gpus_per_task: 1
 
-  - label: "GPU ECCO tests"
-    key: "gpu_ecco_tests"
+  - label: "GPU Datasets tests"
+    key: "gpu_datasets_tests"
     env:
-      TEST_GROUP: "ecco"
+      TEST_GROUP: "datasets"
```

Comment:
will this include JRA55?

Reply:
hm... good point
in an ideal world yes
but perhaps until then should we call this group `ecco_en4`?

Reply:
I renamed it for now.